task_1.py

```diff
@@ -0,0 +1,10 @@
+print("Multiparadigm programming languages : Task 1")
+print("Danil Mashkin, FI-9119")
+print("Vladyslav Valeriyovych, "*44,end = "Vladyslav Valeriyovych. \n" )
+num_1 = 123 - 23.1
+num_2 = 2**3
+num_2 += 2.2
+num_3 = 127.24 - 21.1
+num_4 = 3.2
+num_5 = ((num_1 * num_2) / num_3) - num_4
```

Comment:
It's a good idea to divide the calculation for several lines. but the implementation looks strange for me.
Frankly, now I cannot  provide you an ultimate solution. 
let's discuss, how it could be arranged better at the lab.